pandas/core/dtypes/common.py

```diff
@@ -87,10 +88,10 @@ def ensure_categorical(arr):
     return arr
 
 
-def ensure_int64_or_float64(arr, copy=False):
+def ensure_int_or_float(arr: ArrayLike, copy=False) -> np.array:
```

Comment:
Nice annotation!

Reply:
hah thanks, it was actually due to the other reviewer :)